Dockerfile

```diff
@@ -1,111 +1,47 @@
-#
-# CLA_BACKEND Dockerfile all environments
-#
-# Pull base image.
-FROM phusion/baseimage:0.9.22
-
-# Set correct environment variables.
-ENV HOME /root
-
-# Use baseimage-docker's init process.
-CMD ["/sbin/my_init"]
-
-# Set timezone
-RUN echo $TZ > /etc/timezone && \
-    apt-get update && apt-get install -y tzdata && \
-    rm /etc/localtime && \
-    ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && \
-    dpkg-reconfigure -f noninteractive tzdata && \
-    apt-get clean
-
-# Remove SSHD
-RUN rm -rf /etc/service/sshd /etc/my_init.d/00_regen_ssh_host_keys.sh
-
-# Dependencies
-RUN DEBIAN_FRONTEND='noninteractive' \
-  apt-get update && apt-get install -y \
-    apt-utils \
-    bash \
-    build-essential \
-    g++ \
-    git \
-    libpcre3 \
-    libpcre3-dev \
-    libpq-dev \
-    libxslt-dev \
-    libxml2-dev \
-    make \
-    python-dev \
-    python-pip \
-    python-software-properties \
-    software-properties-common && \
-  apt-get -y build-dep \
-    python-psycopg2 && \
-  rm -rf /var/lib/apt/lists/*
-
-# Install Nginx.
-RUN DEBIAN_FRONTEND='noninteractive' add-apt-repository ppa:nginx/stable && apt-get update
-RUN DEBIAN_FRONTEND='noninteractive' apt-get -y install nginx-full && \
-  chown -R www-data:www-data /var/lib/nginx
-
-ADD ./docker/nginx.conf /etc/nginx/nginx.conf
-ADD ./docker/htpassword /etc/nginx/conf.d/htpassword
-RUN rm -f /etc/nginx/sites-enabled/default && \
-    chown www-data:www-data /etc/nginx/conf.d/htpassword
-
-# Pip install Python packages
+FROM alpine:3.9
+
+RUN apk add --no-cache \
+      bash \
+      gettext \
+      nginx \
+      py2-pip \
+      supervisor \
+      tzdata \
+      uwsgi
+
+# To install pip dependencies
+RUN apk add --no-cache \
+      build-base \
+      git \
+      libxml2-dev \
+      libxslt-dev \
+      postgresql-dev \
+      python2-dev
+
+RUN cp /usr/share/zoneinfo/Europe/London /etc/localtime
 RUN pip install -U setuptools pip==18.1 wheel
-RUN pip install GitPython uwsgi requests
-
-RUN mkdir -p /var/log/wsgi && \
-    touch /var/log/wsgi/app.log /var/log/wsgi/debug.log && \
-    chown -R www-data:www-data /var/log/wsgi && \
-    chmod -R g+s /var/log/wsgi
 
-RUN  mkdir -p /var/log/nginx/cla_backend
-ADD ./docker/cla_backend.ini /etc/wsgi/conf.d/cla_backend.ini
-
-# Define mountable directories.
-VOLUME ["/data", "/var/log/nginx", "/var/log/wsgi"]
```

Comment:
🙋‍♂ I'm unsure if these are still needed, so I removed them.

Reply:
I thought these were needed to persist the apps and webserver logs

Reply:
True, I have no idea how logs are collected from the container

Reply:
Still seems to work:
```
/var/log/nginx$ tail -f demo-backend.cla.dsd.io.access.json
{"timestamp_msec": "1557832907.287", ... }
```
That timestamp is Tuesday, May 14, 2019 12:21:47.287 PM GMT+01:00 DST, so it definitely still gets logged on the Docker host machine. 🤷‍♂

Reply:
Oh... The host machine has an nginx on its own 🤦‍♂

Reply:
```
$ sudo docker exec -ti cla_backend tail -1 /var/log/nginx/cla_backend/access.log
{ "@timestamp": "2019-05-14T12:26:27+01:00", @fields: { "upstream_addr": "unix:////tmp/backend.sock" } }
```

```
$ tail -1 /var/log/nginx/demo-backend.cla.dsd.io.access.json
{"timestamp_msec": "1557833187.286", "@fields": { "upstream_addr": "127.0.0.1:8000" } }
```

Reply:
Is the flow:
ELB->host-nginx->docker-nginx
or
ELB->docker-nginx

Reply:
Yes, there's a host nginx in the middle and the Kibana instance only sees those logs (even historically).

Reply:
It seems we use [beaver](https://python-beaver.readthedocs.io/en/latest/) to ship the logs to logstash, so it eventually shows up in Kibana.

Reply:
*TL;DR* As far as I can see, the `VOLUME` command is safe to remove because it's unused.